### PR TITLE
Add useful buttons for environmental testing

### PR DESCRIPTION
**Pull request by John15321**
### Description

Add useful buttons for environmental testing for steering motors, servos etc

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -788,8 +782,97 @@ function App() {
...
+                                            font: {
+                                                size: 30,
+
+                                            }
+                                        },
+                                        title: {
+                                            display: true,
+                                            font: {
+                                                weight: 'bold',
+                                                size: 40,
```
Comment (jkaszowski) [inline src/App.tsx:844]: dej tu 30

File: src/App.tsx [hunk 2]
```diff
@@ -788,8 +782,97 @@ function App() {
...
+                                            },
+                                            text: "Temperature [C]"
+                                        },
+                                    },
+                                    x: {
+                                        title: {
+                                            display: true,
+                                            font: {
+                                                weight: 'bold',
+                                                size: 40,
```
Comment (jkaszowski) [inline src/App.tsx:854]: tu też

**Review by jkaszowski — APPROVED** [review]
jest ok

File: src/App.tsx (hunk 2 above)
Comment (John15321) [reply src/App.tsx]: done

File: src/App.tsx (hunk 1 above)
Comment (John15321) [reply src/App.tsx]: done

**Pull request merged**